docs/source/guides/network_node/staking_guide.rst

```diff
@@ -512,15 +512,42 @@ This can help to decrease gas consumption in some operations. To merge two stake
 Remove unused sub-stake
 ***********************
 
-Merging or editing sub-stakes can lead to 'unused', inactive sub-stakes remaining on-chain. 
-These unused sub-stakes add unnecessary gas costs to daily operations.
-To remove unused sub-stake:
+When sub-stakes terminate, or after merging or editing sub-stakes,
```

Comment:
```suggestion
When sub-stakes terminate, merge, or are edited there may be there may be 'unused', inactive sub-stakes remaining on-chain.
```

(unsure how to apply suggestion to multiple lines)

Reply:
👍